pytorch_lightning/callbacks/pt_callbacks.py

```diff
@@ -150,20 +160,21 @@ def on_train_end(self, logs=None):
 
 
 class ModelCheckpoint(Callback):
-    """Save the model after every epoch.
-
-    The `filepath` can contain named formatting options,
-    which will be filled the value of `epoch` and
-    keys in `logs` (passed in `on_epoch_end`).
-    For example: if `filepath` is `weights.{epoch:02d}-{val_loss:.2f}.hdf5`,
-    then the model checkpoints will be saved with the epoch number and
-    the validation loss in the filename.
-
-    # Arguments
-        filepath: string, path to save the model file.
-        monitor: quantity to monitor.
-        verbose: verbosity mode, 0 or 1.
-        save_top_k: if `save_top_k == k`,
+    r"""
+
+    Save the model after every epoch.
+
+    Args:
+        filepath (str): path to save the model file.
+            Can contain named formatting options to be auto-filled.
+
+            Example:
+                >>> # save epoch and val_loss in name
+                >>> ModelCheckpoint(filepath='{epoch:02d}-{val_loss:.2f}.hdf5')
+                >>> # saves file like: /path/epoch_2-val_loss_0.2.hdf5
+        monitor (str): quantity to monitor.
```

Comment:
We shall decide what kind of documentation we are going to use...
https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html